barman/backup_executor.py

```diff
@@ -608,6 +615,14 @@ def backup_copy(self, backup_info):
         # for the whole duration of the copy
         self.server.close()
 
+        # Find the backup_manifest file of the previous backup in case it is
+        # an incremental backup
+        parent_backup_manifest = None
```

Comment:
A little suggestion here, if is just a path and not a full manifest, the name of the var should reflect it.
simply put: reading `parent_backup_manifest` I was expecting to get the full contents of the manifest in there, not just the path to the file. so should be `parent_backup_manifest_path`.
Same for the name of the method `_get_backup_manifest_from_backup`, I would expect this to return the full manifest file contents, not just the path.

Reply:
Although the name looks longer, which we usually tend to avoid, it provides a more clear understanding of its purpose.